windows-release/uploadrelease.ps1

```diff
@@ -87,19 +88,28 @@ $dirs = gci "$build" -Directory
 if ($embed) {
     $dirs = ($dirs, (gi $embed)) | %{ $_ }
 }
+if ($sbom) {
+    $dirs = ($dirs, $sbom)
+}
 
 foreach ($a in $dirs) {
     "Uploading files from $($a.FullName)"
     pushd "$($a.FullName)"
     $exe = gci *.exe, *.exe.asc, *.zip, *.zip.asc
     $msi = gci *.msi, *.msi.asc, *.msu, *.msu.asc
+    $spdx_json = gci *.spdx.json
     popd
 
     if ($exe) {
         & $pscp -batch -hostkey $hostkey -noagent -i $keyfile $exe.FullName "$user@${server}:$d"
         if (-not $?) { throw "Failed to upload $exe" }
     }
 
+    if ($spdx_json) {
+        & $pscp -batch -hostkey $hostkey -noagent -i $keyfile $spdx_json.FullName "$user@${server}:$d"
+        if (-not $?) { throw "Failed to upload $spdx_json" }
```

Comment:
```suggestion
        if (-not $?) { Write-Host "##[warning]Failed to upload $spdx_json" }
```

At least until we're confident this is going to work, I'd rather not break the next release, especially after files have already been uploaded. All a break here does is make it highly likely that we'll end up with a second build and mismatched binaries between sources (e.g. Nuget will have already published by this point).